### ISSUE TITLE
Pre-train link error

### ISSUE THREAD

**Issue by Annguyen1512** [alinvdu/reproduce-dream-diffusion#5]
Hi! I have checked and found that the Google Drive link in README.md is no longer available. Can you please update the new link again? Thank you very much and I hope you respond soon.

**Comment by alinvdu** [comment]
Here you can find a model for predicting EEG signal: https://drive.google.com/file/d/1G79aAmd2pO4ZPuu9FC6dI_A4gqmAuiJu/view?usp=sharing trained for 500 epochs with different masking ratio - I updated the link in the Readme as well.

**Comment by Annguyen1512** [comment]
Thank you so much
